src/index.scss

```diff
@@ -69,13 +75,18 @@
   --srijan4-gradient-11: linear-gradient(180deg, #b621fe 0%, #1fd1f9 100%);
 
   --srijan4-team-gradient: linear-gradient(161.91deg, #010044 0%, #02014c 66.54%);
+  --srijan4-gradient-12: linear-gradient(161.91deg, #02010d 0%, #040344 66.54%);
+  --srijan4-gradient-13: linear-gradient(180.56deg, #0704b9 -22.26%, #b4015f 159.83%),
+    linear-gradient(0deg, #000000, #000000);
 
   --heading-1-text-weight: 600;
   --heading-2-text-weight: 500;
   --heading-3-text-weight: 700;
 
   --paragraph-1-text-weight: 400;
   --paragraph-2-text-weight: 300;
+
+  --srijan4-navbar-height: 4vh;
```

Comment:
Use this height for Navbar's height in Navbar.module.scss so that the concerned person would know to use this height